dataprofiler/labelers/base_data_labeler.py

```diff
@@ -914,4 +914,4 @@ def load_with_components(
         data_labeler.set_preprocessor(preprocessor)
         data_labeler.set_model(model)
         data_labeler.set_postprocessor(postprocessor)
-        return data_labeler
+        return cast(TrainableDataLabeler, data_labeler)
```

Comment:
Hmm, should we allow for `TrainableDataLabeler` or `CustomTrainableDataLabeler`?

Reply:
or does that cause downstream issues?

Reply:
issue is that the `CustomTrainableDataLabeler` type only exists locally so it wouldn't be a valid return type.